airflow/decorators/python.py

```diff
@@ -39,12 +39,12 @@ class _PythonDecoratedOperator(DecoratedOperator, PythonOperator):
     :type multiple_outputs: bool
     """
 
-    template_fields = ('op_args', 'op_kwargs')
+    template_fields: Sequence[str] = ('op_args', 'op_kwargs')
```

Comment:
Won't `Tuple[str, ...]` be more proper type?
> To specify a variable-length tuple of homogeneous type, use literal ellipsis, e.g. Tuple[int, ...]. 

To me `Sequence` is ambiguous type.

Reply:
The problem with TemplateFields as type is that we will have to import it from somewhere in Providers (and that would either make them incompatible with pre-Airlfow 2.3 or we will have to `quote` those and add if TYPE_CHECKING trick as we as we did with Context. In this case Sequence[str] sounds like good compromise (especially that till recently we recommended Lists in our examples and Tuple would make anyone's custom operator immediately failing MyPy check (which is not really good in "mass").  I think our intent is a bit different:

* we still accept LIst[] (which is a Sequence as well)
* but in our operators we implement this always as a Tuple for its immutability
* so if you want in your custom operator you can still use List if you want